src/main/webapp/WEB-INF/views/car/add_driver_to_car.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+    <h1>Please fill the form bellow</h1>
+    <form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+        Driver_id <input type="text" name="driver_id"><br>
```

Comment:
why type text if you input integers and make both fields required

Reply:
because of copy past